### DISCUSSION TITLE
Problems making GeNN work

### DISCUSSION THREAD

**Discussion by bjoernapfel** [genn-team/genn#549] [GeNN questions]
After having installed GeNN (windows, CPU only) and wanting to run the quickstart (compiling additional tools worked), I got the same error on two different laptops
![thumbnail_589693D01FBD4EA0B36540C5B9DAD9D7](https://user-images.githubusercontent.com/116563213/197520394-9fd4c196-aab2-4677-b4e7-f02e8e897760.png)
Running the python tutorials also failed as no module named pygenn could be found. There clearly is some issue but I can´t wrap my head around what could cause it, any ideas?

**Comment by tnowotny** [comment]
Hi, this is interesting. So, status -1073741819 on Windows apparently is the return value for when a segmentation fault occurred. We are very confident that there are no memory access errors in the standard example you tried to run, so there must be a different underlying issue. 
Maybe we can make some progress with respect to pyGeNN first.
Have you followed the instructions "Installing PyGeNN from source on Windows" on the page "Installation" https://genn-team.github.io/genn/documentation/4/html/d8/d99/Installation.html ?
If so, were there any errors?

**Comment by neworderofjamie** [comment]
Embarrassingly, @tnowotny was overly confident in the battle-testedness of the standard example. In https://github.com/genn-team/genn/pull/551, I just fixed a bug I introduced almost 4 years ago which was causing this very segmentation fault!

**Comment by tnowotny** [comment]
Yeah - I need to eat humble pie there. Battle-tested but modernisation sunk it.
